src/renderer/pages/PlayerPage.js

```diff
@@ -109,7 +109,7 @@ function renderMedia (state) {
   var MediaTagName = state.playing.type
   var mediaTag = (
     <MediaTagName
-      src={state.server.localURL}
+      src={state.server.localURL + '/' + state.playing.fileIndex}
```

Comment:
How was this working before, given that you can click Play on an individual file in a torrent? (Did `localURL` previously contain the file index, but it doesn't anymore?)